clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -162,6 +162,11 @@
     public static final String REQUEST_TIMEOUT_MS_CONFIG = CommonClientConfigs.REQUEST_TIMEOUT_MS_CONFIG;
     private static final String REQUEST_TIMEOUT_MS_DOC = CommonClientConfigs.REQUEST_TIMEOUT_MS_DOC;
 
+    /** <code>interceptor.classes</code> */
+    public static final String INTERCEPTOR_CLASSES_CONFIG = "interceptor.classes";
+    public static final String INTERCEPTOR_CLASSES_DOC = "A list of classes to use as interceptors. "
+                                                        + "Implementing the <code>ConsumerInterceptor</code> interface allows plugging in classes that can intercept records "
+                                                        + "at different points on consumer, and, in some cases, mutate the records. By default, there are no interceptors.";
```

Comment:
Minor: this sentence reads a little awkward:
```
Implementing the ConsumerInterceptor interface allows plugging in classes that can intercept records at different points on consumer...
```
Maybe just get rid of the "allows plugging in classes"? For example:
```
Implementing the ConsumerInterceptor interface allows you to intercept (and possibly mutate) records received by the consumer.
```